.ci_support/build_all.py

```diff
@@ -52,7 +52,7 @@ def build_all(recipes_dir, arch):
     if found_cuda:
         print('##vso[task.setvariable variable=NEED_CUDA;isOutput=true]1')
```

Comment:
Do we need to change this to use `os.environ` as well?

Reply:
No, we still have a separate job for cuda